src/LoadableModule.h

```diff
@@ -9,26 +9,26 @@
 #ifndef SQUID_SRC_LOADABLEMODULE_H
 #define SQUID_SRC_LOADABLEMODULE_H
 
-#include "SquidString.h"
+#include "sbuf/SBuf.h"
 
 // wrapper for dlopen(3), libltdl, and friends
 class LoadableModule
 {
 public:
-    LoadableModule(const String &aName);
+    LoadableModule(const SBuf &aName);
```

Comment:
If possible without heroic efforts, let's block unwanted implicit conversions:

```suggestion
    explicit LoadableModule(const SBuf &aName);
```